### PR TITLE
Fixes 0 stride handling

### LINKED ISSUE #174: SSX Tricky - Totally messed up graphics
Using Complex_4627.bin, mcpx_1.0.bin on an Intel(R) Core(TM) i7-8559U CPU @ 2.70GHz with integrated Intel® Iris® Plus Graphics 655. Running on ubuntu 20.10.
QEMU emulator version 5.1.0 (Debian 0.5.0-14-gd9046372c7-0~202101280232~ubuntu20.10.1)

https://user-images.githubusercontent.com/5549373/106322790-a7741e00-6276-11eb-92e9-5445170a91c7.mp4

### LINKED ISSUE #686: SSX Tricky: Stage select missing foreground graphics
### Title

https://xemu.app/titles/45410004/#SSX-Tricky

### Bug Description

The foreground of the stage select screen is not rendered at all

Screenshot_20220128_083040

### Expected Behavior

(Please ignore the glare in the upper left corner, that's an external reflection and not rendered)

ssx_tricky_expected_select

### xemu Version

Version: 0.6.2-53-g6b821d2062
Branch: master
Commit: 6b821d2
Date: Mon Jan 24 09:41:21 PM UTC 2022

### System Information

OS: Kubuntu 21.04 - 5.11.0-38-generic
CPU: Core i7-6700K @ 4GHz
GPU: NVIDIA GeForce GTX 1070
GPU Driver: 4.6.0 NVIDIA 470.74

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by abaire**
Special cases attributes with stride = 0 by using the first value in the buffer for all vertices, matching hardware.

[Test case](https://github.com/abaire/nxdk_pgraph_tests/blob/main/src/tests/zero_stride_tests.cpp)
[Results](https://github.com/abaire/nxdk_pgraph_tests_golden_results/wiki/Results-Zero_stride)

Fixes #686
Fixes #174

### CODE REVIEW

File: hw/xbox/nv2a/pgraph.c [hunk 1]
```diff
@@ -6685,11 +6685,15 @@ static void pgraph_bind_vertex_attributes(NV2AState *d,
...
             last_entry += stride * provoking_element_index;
+            pgraph_update_inline_value(attr, last_entry);
         } else {
-            last_entry += element_size * provoking_element_index;
+            // Stride of 0 indicates that only the first element should be
+            // used.
+            pgraph_update_inline_value(attr, last_entry);
+            glDisableVertexAttribArray(i);
+            glVertexAttrib4fv(i, attr->inline_value);
+            continue;
```
Comment (mborgerson) [inline hw/xbox/nv2a/pgraph.c:6681]: As written, this `continue` is unnecessary. However, we could also move this last entry update block up and eliminate some GL calls in the process (i.e. we don't need to set attrib pointer and enable it just to then disable it)

Comment (abaire) [reply hw/xbox/nv2a/pgraph.c:6681]: Good call, updated.

**Pull request merged**